### PR TITLE
Added translation to Russian

### PR DESCRIPTION
**Pull request by OIgnat**
Comments to this translation will be published here inside discussion branch: https://github.com/Joystream/community-repo/issues/212

Discord user name: AlexZNet#0968 (for any feedbacks or requests for fixes depending on below points or any other reasons).

1. Some terms, like 'Blockchain developer' that contain well-known definitions are translated leaving those definitions as they are. For example 'Blockchain разработчик'. It is usually called in this way locally and will not make any confusion, but looks familiar to everyone. However, if fully native translation looks better, this could be changed.
1.1. at the same time, 'blockchain' and similar definitions were translated.
1.2. definitions like 'testnet' are used as transliterations, because they are commonly used in community and could be understand better than translating them as 'test network' value. Also, definition is simple enough to be understand even for those who do not know, what 'testnet' is.

2. Unlike English, in Russian there are a couple of ways of addressing people, who read text. Let's imagine, as example 'You' and 'you' (can't be expressed as simple, though) - the last one is more modern and simple approach, that, to my mind, fit Joystream concept, according to what I have read at different pages like branding or about Joustream. But if you think the 1-st, more formal one, is better - I can change that.

3. There are some blocks of text like 'day' and 'days'. In Russian there multiple possible values for multiples - but applying this would need technical improvements on website. So multiples were used in the most common use cases.

4. The names of testnets are left in Latin - because they are original names. At the same time, when used in the meaning of city (like Rome as a city) - they were translated.

5. After setting up website locally and checking how it looks like after translation I noticed that 'Road to mainnet' on landing page is not changed, even being translated in json file. So looks like a hardcoded value.

6. At brand page there are definitions like 'Joysteram Blue' & Gray. As I get, those are namings, so they were left as definitions.

### CODE REVIEW

**Comment by OIgnat** [comment]
Added Ukrainian translation

File: src/locales/ru/token.json [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
+        "title": "Как можно получить tJOY?",
+        "text": "Есть много возможностей получить tJOY. Меньшие суммы можно запросить на наших каналах сообщества (включая наш сервер Discord и форум), а большие суммы можно накопить, участвуя в грантах, конкурсах и других мероприятиях."
       },
       "linkBetweenTJoyAndFutureToken": {
-        "title": "Is there any link between tJOY and the future mainnet token?",
-        "text": "There is no direct link between tJOY and mainnet JOY. We are using tJOY to simulate the real economic incentives present on mainnet to aid the platform's development and facilitate the growth and education of our community in preparation of the transition from testnet to mainnet."
+        "title": "Существует ли связь между tJOY и будущими токенами основной сети?",
+        "text": "Нет прямой связи между tJOY и JOY. Мы используем tJOY для моделирования реальных экономических стимулов, присутствующих в основной сети, чтобы способствовать развитию платформы и способствовать росту и обучению нашего сообщества при подготовке перехода от тестнета к реальной работе."
```
Comment (singulart) [inline src/locales/ru/token.json:37]: к реальной работе -> к основной сети

File: src/locales/ru/token.json [hunk 2]
```diff
@@ -1,67 +1,67 @@
...
       },
       "howToCashout": {
-        "title": "How do I cashout tJOY to USD?",
-        "text": "We use Bitcoin Cash (BCH) as a payment rail on our testnets. You can cashout tJOY by making a transfer to the dedicated burn address: <0>5D5PhZQNJzcJXVBxwJxZcsutjKPqUPydrvpu6HeiBfMaeKQu</0> Before doing this, you should set the <0>memo</0> of the key from which the transfer is made to your BCH address. At the moment of the transfer, the BCH to USD exchange rate is locked. <1/><1/>More information on payouts can be found <2>here</2>."
+        "title": "Как обналичить tJOY в USD?",
+        "text": "Мы используем Bitcoin Cash (BCH) в качестве средства оплаты в наших тестнетах. Вы можете обналичить tJOY, сделав перевод на выделенный адрес: <0>5D5PhZQNJzcJXVBxwJxZcsutjKPqUPydrvpu6HeiBfMaeKQu</0> Перед тем как это сделать, вы должны установить <0>memo</0> ключа, с которого осуществляется перевод на ваш адрес BCH. Мы фиксируем курс BCH/USD на момент осуществления перевода. <1/><1/>С детальной информацией можно ознакомиться <2>здесь</2>."
```
Comment (singulart) [inline src/locales/ru/token.json:41]: средства оплаты -> криптовалюты для вывода средств

File: src/locales/ru/token.json [hunk 3]
```diff
@@ -1,67 +1,67 @@
...
       },
       "howToEarn": {
-        "title": "How can I earn mainnet JOY?",
-        "text": "Currently, the only way to secure an allocation of mainnet JOY is through high-quality contributions to the platform and community via our <0>Founding Member Program</0>."
+        "title": "Как можно заработать токены основной сети JOY?",
+        "text": "В настоящее время единственный способ обеспечить распределение JOY в основной сети - это качественный вклад в платформу и сообщество через нашу <0>Программу для членов-учредителей</0>."
```
Comment (singulart) [inline src/locales/ru/token.json:45]: ключевых участников

File: src/locales/ru/token.json [hunk 4]
```diff
@@ -1,67 +1,67 @@
...
-        "text": "You can check the live exchange rate (tJOY->USD) on this page (see above)."
+        "title": "Сколько стоит tJOY?",
+        "text": "Курс обмена (tJOY->USD) можно посмотреть на этой странице (см. выше)."
       }
     },
     "cashouts": {
-      "title": "Pending cashouts",
-      "subtitle": "Tokens sent to Jsgenesis in this exchange are burned, hence the final price of the token is unaffected by such an exchange.",
+      "title": "Ожидаемые выплаты",
+      "subtitle": "Токены, отправленные Jsgenesis на обмен, будут утилизированы, т.к. конечная стоимость токенов должна остаться неизменной при таком обмене.",
```
Comment (singulart) [inline src/locales/ru/token.json:54]: утилизированы -> сожжены
т.к. -> поэтому

File: src/locales/ru/sumer.json [hunk 5]
```diff
@@ -1,47 +1,47 @@
...
+          "title": "Сделать доступным публикацию контента из интерфейса для воспроизведения.",
+          "text": "Sumer представляет возможность создавать каналы и видео и управлять ими в Joystream Player, заменяя громоздкий и сложный процесс публикации, который требует работы с интерфейсом командной строки, которым необходимо было пользоваться в теснете Antioch."
         },
         "deployingNewContentDirectory": {
-          "title": "Deploying the new content directory",
-          "text": "As part of Sumer we will be introducing a simple native runtime module. This is because technical complexities and tradeoffs associated with the previous iterations of the content directory were not acceptable."
+          "title": "Публикация нового каталога данных",
+          "text": "В рамках Sumer мы предоставляем простой встроенный модуль времени выполнения. Это связано с тем, что технические сложности и необходимость компромиссов, в предыдущих итерациях развития каталога были неприемлемы."
```
Comment (singulart) [inline src/locales/ru/sumer.json:31]: простой встроенный модуль времени выполнения -> исправленный системный компонент по управлению контентом

итерациях развития -> версиях

File: src/locales/ru/sumer.json [hunk 6]
```diff
@@ -1,47 +1,47 @@
...
         },
         "enableBuilderRole": {
-          "title": "Enable builder/platform operations role",
-          "text": "We are introducing a new working group focused on platform operations. Originally, we had also planned to enable the new \"Gateway\" role, though this is no longer in the scope of the Sumer release."
+          "title": "Подключения роли строителя платформы",
+          "text": "Мы представляем новую рабочую группу, специализирующуюся на работе с платформамой. Первоначально мы также планировали включить новую роль \"Шлюз\", хотя это не вошло в выпуск Sumer."
```
Comment (singulart) [inline src/locales/ru/sumer.json:35]: специализирующуюся на работе с платформамой -> специализирующуюся на функциональных улучшениях платформы

File: src/locales/ru/sumer.json [hunk 7]
```diff
@@ -1,47 +1,47 @@
...
         "deployingNewContentDirectory": {
-          "title": "Deploying the new content directory",
-          "text": "As part of Sumer we will be introducing a simple native runtime module. This is because technical complexities and tradeoffs associated with the previous iterations of the content directory were not acceptable."
+          "title": "Публикация нового каталога данных",
+          "text": "В рамках Sumer мы предоставляем простой встроенный модуль времени выполнения. Это связано с тем, что технические сложности и необходимость компромиссов, в предыдущих итерациях развития каталога были неприемлемы."
         },
         "enableBuilderRole": {
-          "title": "Enable builder/platform operations role",
-          "text": "We are introducing a new working group focused on platform operations. Originally, we had also planned to enable the new \"Gateway\" role, though this is no longer in the scope of the Sumer release."
+          "title": "Подключения роли строителя платформы",
```
Comment (singulart) [inline src/locales/ru/sumer.json:34]: Добавить роль "Оператор платформы"

File: src/locales/ru/sumer.json [hunk 8]
```diff
@@ -1,47 +1,47 @@
...
-      "title": "Testnet Goals",
-      "subtitle": "The goals listed below are a simplified representation of our main objectives for the Sumer testnet.",
+      "title": "Цели Тестнета",
+      "subtitle": "Приведенные ниже цели представляют собой упрощенное представление ключевых результатов тестнета Sumer",
       "goals": {
         "contentPublishing": {
-          "title": "Enable content publishing from playback interface",
-          "text": "Sumer introduces the ability to create and manage channels and videos in the Joystream Player, replacing the cumbersome and technically challenging CLI-dependent process required on the Antioch testnet."
+          "title": "Сделать доступным публикацию контента из интерфейса для воспроизведения.",
+          "text": "Sumer представляет возможность создавать каналы и видео и управлять ими в Joystream Player, заменяя громоздкий и сложный процесс публикации, который требует работы с интерфейсом командной строки, которым необходимо было пользоваться в теснете Antioch."
```
Comment (singulart) [inline src/locales/ru/sumer.json:27]: теснете - опечатка

File: src/locales/ru/sumer.json [hunk 9]
```diff
@@ -1,47 +1,47 @@
...
+      "text": "Среди других изменений выпуск тестнета Sumer предлагает значительные улучшения в процессе загрузки контента через наше приложение для воспроизведения."
     },
     "criticalDocuments": {
-      "title": "Critical Documents",
+      "title": "Важная документация",
       "releasePlan": {
-        "title": "Release Plan",
-        "text": "Read the release plan as it was made during the planning stage, and learn more about how the development evolved."
+        "title": "Релиз план",
+        "text": "Изучи релиз план в том виде, каким он был создан во время планирования и узнавай, как проходила разработка."
```
Comment (singulart) [inline src/locales/ru/sumer.json:14]: Изучите, узнайте

File: src/locales/ru/sumer.json [hunk 10]
```diff
@@ -1,47 +1,47 @@
...
       },
       "blogPost": {
-        "title": "Announcement Blog Post",
-        "text": "Read a brief primer on the Sumer testnet and its objectives."
+        "title": "Объявление в блоге",
+        "text": "Изучи краткое руководство по тестнету Sumer и его целям."
```
Comment (singulart) [inline src/locales/ru/sumer.json:18]: Изучить

File: src/locales/ru/sumer.json [hunk 11]
```diff
@@ -1,47 +1,47 @@
...
         "contentPublishing": {
-          "title": "Enable content publishing from playback interface",
-          "text": "Sumer introduces the ability to create and manage channels and videos in the Joystream Player, replacing the cumbersome and technically challenging CLI-dependent process required on the Antioch testnet."
+          "title": "Сделать доступным публикацию контента из интерфейса для воспроизведения.",
+          "text": "Sumer представляет возможность создавать каналы и видео и управлять ими в Joystream Player, заменяя громоздкий и сложный процесс публикации, который требует работы с интерфейсом командной строки, которым необходимо было пользоваться в теснете Antioch."
         },
         "deployingNewContentDirectory": {
-          "title": "Deploying the new content directory",
-          "text": "As part of Sumer we will be introducing a simple native runtime module. This is because technical complexities and tradeoffs associated with the previous iterations of the content directory were not acceptable."
+          "title": "Публикация нового каталога данных",
```
Comment (singulart) [inline src/locales/ru/sumer.json:30]: Каталога контента

File: src/locales/ru/acropolis.json [hunk 12]
```diff
@@ -1,59 +1,59 @@
...
     "siteMetadata": {
-      "description": "Explore the Acropolis testnet"
+      "description": "Изучаем тестнет Acropolis"
     },
     "hero": {
-      "title": "Acropolis Network",
-      "text": "Explore available roles and pick the one that suits you best. Influence the platform's development and earn real money in the process.",
-      "chipText": "What is this?"
+      "title": "Сеть Acropolis",
+      "text": "Изучай доступные варианты участия и выбирай те, которые тебе больше всего подходят. Влияй на развитие платформы и зарабатывай при этом реальные деньги.",
```
Comment (singulart) [inline src/locales/ru/acropolis.json:8]: Изучить

File: src/locales/ru/acropolis.json [hunk 13]
```diff
@@ -1,59 +1,59 @@
...
+      "text": "<0>Известный своей великолепной архитектурой, Акрополь</0> возможно, является самым знаменитым строением Парфенона. Он был построен в честь победы над персидскими завоевателями, и сегодня является символом демократии и западной цивилизации."
     },
     "criticalDocuments": {
-      "title": "Critical Documents",
+      "title": "Важная документация",
       "fullSpecifications": {
-        "title": "Full Specifications",
-        "text": "Read the specs of the newly implemented features of Acropolis."
+        "title": "Полная спецификация",
+        "text": "Читай спецификацию новых возможностей Acropolis."
```
Comment (singulart) [inline src/locales/ru/acropolis.json:22]: Прочесть

File: src/locales/ru/acropolis.json [hunk 14]
```diff
@@ -1,59 +1,59 @@
...
       },
       "releasePlan": {
-        "title": "Release Plan",
-        "text": "Read the release plan as it was made during the planning stage, and learn more about how the development evolved."
+        "title": "Релиз план",
+        "text": "Читай релиз план в том виде, каким он был создан во время планирования и узнавай, как проходила разработка."
```
Comment (singulart) [inline src/locales/ru/acropolis.json:26]: Прочесть

File: src/locales/ru/alexandria.json [hunk 15]
```diff
@@ -1,55 +1,55 @@
...
+      "text": "В Египте, на месте Столпа Помпея в Александрии, распложено два сфинкса. Сфинксы и связанная с ними колонна - один из самых важных древних памятников Александрии.<0/><0/> Александрийских сфинксов не следует путать с более известным Большим Сфинксом, находящимся в Гизе, чуть менее 200 километров к юго-востоку от Александрии."
     },
     "criticalDocuments": {
-      "title": "Critical Documents",
+      "title": "Важная документация",
       "releasePlan": {
-        "title": "Release Plan",
-        "text": "Read the release plan as it was made during the planning stage, and learn more about how the development evolved."
+        "title": "Релиз план",
+        "text": "Читай релиз план в том виде, каким он был создан во время планирования и узнавай, как проходила разработка."
```
Comment (singulart) [inline src/locales/ru/alexandria.json:22]: Прочесть

File: src/locales/ru/alexandria.json [hunk 16]
```diff
@@ -1,55 +1,55 @@
...
       },
       "blogPost": {
-        "title": "Announcement Blog Post",
-        "text": "Read a brief primer on the Alexandria testnet and its objectives."
+        "title": "Объявление",
+        "text": "Читай краткое руководство по тестнету Alexandria и его целям."
```
Comment (singulart) [inline src/locales/ru/alexandria.json:26]: Прочесть

File: src/locales/ru/antioch.json [hunk 17]
```diff
@@ -1,42 +1,42 @@
...
+      "title": "ЗАМЕЩЕН SUMER "
     },
     "criticalDocuments": {
-      "title": "Critical Documents",
+      "title": "Важная документация",
       "releasePlan": {
-        "title": "Release Plan",
-        "text": "Read the release plan as it was made during the planning stage, and learn more about how the development evolved."
+        "title": "Релиз план",
+        "text": "Читай релиз план в том виде, каким он был создан во время планирования и узнавай, как проходила разработка."
```
Comment (singulart) [inline src/locales/ru/antioch.json:17]: Прочесть

File: src/locales/ru/antioch.json [hunk 18]
```diff
@@ -1,42 +1,42 @@
...
       },
       "blogPost": {
-        "title": "Announcement Blog Post",
-        "text": "Read a brief primer on the Antioch testnet and its objectives."
+        "title": "Объявление",
+        "text": "Читай краткое руководство по тестнету Antioch и его целям."
```
Comment (singulart) [inline src/locales/ru/antioch.json:21]: Прочесть

File: src/locales/ru/athens.json [hunk 19]
```diff
@@ -1,59 +1,59 @@
...
     "siteMetadata": {
-      "description": "Explore the Athens testnet"
+      "description": "Изучаем тестнет Athens"
     },
     "hero": {
-      "title": "Athens Network",
-      "text": "Explore available roles and pick the one that suits you best. Influence the platform's development and earn real money in the process.",
-      "chipText": "What is this?"
+      "title": "Сеть Athens",
+      "text": "Изучай доступные варианты участия и выбирай те, который тебе больше всего подходят. Влияй на развитие платформы и зарабатывай при этом реальные деньги.",
```
Comment (singulart) [inline src/locales/ru/athens.json:8]: Изучите, выберите, вам, Влияйте, зарабатывайте

File: src/locales/ru/athens.json [hunk 20]
```diff
@@ -1,59 +1,59 @@
...
-        "title": "Full Specifications",
-        "text": "No specifications was published for Athens."
+        "title": "Полные характеристики",
+        "text": "Характеристики к тестнету Athens не были опубликованы."
       },
       "releasePlan": {
-        "title": "Release Plan",
-        "text": "Read the release plan as it was made during the planning stage, and learn more about how the development evolved."
+        "title": "Релиз план",
+        "text": "Читай релиз план в том виде, каким он был создан во время планирования и узнавай, как проходила разработка."
```
Comment (singulart) [inline src/locales/ru/athens.json:26]: Читайте, узнавайте

File: src/locales/ru/babylon.json [hunk 21]
```diff
@@ -1,55 +1,55 @@
...
+      "text": "История Вавилонской башни - это миф о происхождении, призванный объяснить, почему народы мира говорят на разных языках.<0/><0/>Согласно этой истории, единая человеческая раса в следующих поколениях после Великого Потопа, говорящая на одном языке и мигрирующая на восток, приходит в землю Сеннаар. Там они собирались построить город с башней, достаточно высокой, чтобы достичь небес.<0/><0/> Бог же, увидев город и башню, путает речь всех людей так, чтобы они больше не могли понимать друг друга, и разбрасывает их по всему миру."
     },
     "criticalDocuments": {
-      "title": "Critical Documents",
+      "title": "Важная документация",
       "releasePlan": {
-        "title": "Release Plan",
-        "text": "Read the release plan as it was made during the planning stage, and learn more about how the development evolved."
+        "title": "Релиз план",
+        "text": "Читай релиз план в том виде, каким он был создан во время планирования и узнавай, как проходила разработка."
```
Comment (singulart) [inline src/locales/ru/babylon.json:22]: Читайте, узнавайте

File: src/locales/ru/babylon.json [hunk 22]
```diff
@@ -1,55 +1,55 @@
...
       },
       "blogPost": {
-        "title": "Announcement Blog Post",
-        "text": "Read a brief primer on the Babylon testnet and its objectives."
+        "title": "Объявление в Блоге",
+        "text": "Читай краткое руководство по тестнету Babylon и его целям."
```
Comment (singulart) [inline src/locales/ru/babylon.json:26]: Прочесть

File: src/locales/ru/constantinople.json [hunk 23]
```diff
@@ -1,51 +1,51 @@
...
+      "text": "Колонна Константина, также известная как Сожженный камень или Сожженный столб, представляет собой римскую монументальную колонну, построенную по приказу римского императора Константина Великого в 330 году н.э. Возведена в честь провозглашения Византии новой столицей Римской империи."
     },
     "criticalDocuments": {
-      "title": "Critical Documents",
+      "title": "Важная документация",
       "releasePlan": {
-        "title": "Release Plan",
-        "text": "Read the release plan as it was made during the planning stage, and learn more about how the development evolved."
+        "title": "Релиз план",
+        "text": "Читай релиз план в том виде, каким он был создан во время планирования и узнавай, как проходила разработка."
```
Comment (singulart) [inline src/locales/ru/constantinople.json:22]: Читайте, узнавайте

File: src/locales/ru/rome.json [hunk 24]
```diff
@@ -1,59 +1,59 @@
...
     "siteMetadata": {
-      "description": "Explore the Rome testnet"
+      "description": "Изучаем тестнет Rome"
     },
     "hero": {
-      "title": "Rome Network",
-      "text": "Explore available roles and pick the one that suits you best. Influence the platform's development and earn real money in the process.",
-      "chipText": "What is this?"
+      "title": "Сеть Rome",
+      "text": "Изучай доступные варианты участия и выбирай те, которые тебе больше всего подходят. Влияй на развитие платформы и зарабатывай при этом реальные деньги.",
```
Comment (singulart) [inline src/locales/ru/rome.json:8]: Изучите, выберите, вам, Влияйте, зарабатывайте

**Review by singulart — CHANGES_REQUESTED** [review]
Left some comments

**Review by singulart — CHANGES_REQUESTED** [review]
Left some comments

File: src/locales/ua/about.json [hunk 25]
```diff
@@ -0,0 +1,51 @@
+{
```
[the remark this answers is not in the document]
Comment (OIgnat) [reply src/locales/ua/about.json:1]: В ru варианте были плейсхолдер-файлы с английским переводом, поэтому в коммите видно английский-русский в виде изменений в файлах. Украинского перевода не было изначально и это новая папка (ua), которую я добавил, поэтому контент просто виден сам по себе (не с чем сравнивать). Чтобы были отображены изменения можно попробовать сделать коммит папки ua с английским вариантом, а потом поверх него - украинский. Не уверен, что в рамках пулл реквеста сработает, но можно попробовать. Напишите, пожалуйста, как стоит поступить.

File: src/locales/ru/sumer.json (hunk 7 above)
Comment (OIgnat) [reply src/locales/ru/sumer.json:34]: Судя по контексту предыдущих правок, возможно использовать "Разработчик платформы"?

**Comment by OIgnat** [comment]
Requested changes were applied.
Also I have created 2 commits for Ukrainian translation. 1-st switches uk folder to English files and 2-nd returns Ukrainian back. The purpose of this was to try to make Ukrainian translation being side-by-side with English, so review could be done easier.

**Review by singulart — COMMENTED** [review]
I have to reject this submission, because the low-effort google e-translations like this one violate the terms of the Bounty.

**Pull request closed without merging**